### PR TITLE
Removed Docker Builds

### PR DESCRIPTION
**Pull request by srivarra**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Docker is no longer necessary for Ark Analysis. Windows users can now completely use Ark Analysis without Docker or on WSL.

**How did you implement your changes**

This PR removes the `Dockerfile` and docker workflows from the repository, along with documentation for how to use Ark with Docker.
**Remaining issues**

N/A

### CODE REVIEW

File: docs/_rtd/faq.md [hunk 1]
```diff
@@ -1,26 +1,16 @@
 ## FAQ
 
-### When I run the notebook, it says "No module named 'ark'"
-
-This error generally happens if you run the notebook outside of Docker. Simply running `jupyter notebook` or `jupyter lab` on a notebook will not start Docker for you. You need to explicitly run `bash start_docker.sh` and follow the prompts in our [README](https://github.com/angelolab/ark-analysis/blob/main/README.md). 
-
-If you continue to run into issues, make sure you don't have any other Docker sessions running. 
-
```
Comment (camisowers) [inline docs/_rtd/faq.md:8]: We could probably point out that this is likely an issue with the ark_env and to ensure ark_env is activated before opening jupyter, or to try re-installing.

**Review by camisowers — APPROVED** [review]
Looks great! One minor suggestion.

**Review by alex-l-kong — CHANGES_REQUESTED** [review]
One additional section that could be removed.

File: docs/_rtd/windows_setup.md [hunk 2]
```diff
@@ -4,12 +4,12 @@ Our repo works runs best on Linux-based systems (which includes Mac OS). If you
 
 ### WSL 2
 
-In order to download Docker, you need to install WSL 2 if you don't already have it. **We find it best to download and configure WSL 2 before anything else**. We recommend using the [Command Prompt](https://en.wikipedia.org/wiki/Cmd.exe) to run these commands (and later ones). You can find it by typing `Command Prompt` in the Windows search bar. 
+**We find it best to download and configure WSL 2 before anything else**. We recommend using the [Command Prompt](https://en.wikipedia.org/wiki/Cmd.exe) to run these commands (and later ones). You can find it by typing `Command Prompt` in the Windows search bar.
```
[the remark this answers is not in the document]
Comment (srivarra) [reply docs/_rtd/windows_setup.md:7]: Yeah I think this is reasonable, if anyone wants to run it on WSL2 they probably already have the know how to set it up themselves.

**Review by alex-l-kong — APPROVED** [review]
👍

**Pull request merged**